plugins/Insights/lang/en.json

```
@@ -1,35 +1,35 @@
         "DayComparedToPreviousDay": "previous day",
-        "DayComparedToPreviousWeek": "same day in the previous week",
-        "DayComparedToPreviousYear": "same day in the previous year",
+        "DayComparedToPreviousWeek": "same day the week before",
+        "DayComparedToPreviousYear": "same day the year before",
         "Filter": "Filter",
-        "FilterIncreaserAndDecreaser": "Increaser & decreaser",
-        "FilterOnlyDecreaser": "Only decreaser",
+        "FilterIncreaserAndDecreaser": "Increasing and decreasing",
+        "FilterOnlyDecreaser": "Only decreasing",
         "FilterOnlyDisappeared": "Only disappeared",
-        "FilterOnlyIncreaser": "Only increaser",
+        "FilterOnlyIncreaser": "Only increasing",
         "FilterOnlyMovers": "Only movers",
         "FilterOnlyNew": "Only new",
         "IgnoredChanges": "Changes affecting less than %s visits were ignored.",
         "MonthComparedToPreviousMonth": "previous month",
-        "MonthComparedToPreviousYear": "same month in the previous year",
-        "MoversAndShakersWidgetTitle": "Movers and Shakers",
-        "NoResultMatchesCriteria": "No rows match the criteria",
+        "MonthComparedToPreviousYear": "same month the year before",
+        "MoversAndShakersWidgetTitle": "Movers and shakers",
+        "NoResultMatchesCriteria": "There are no matching rows",
         "OverviewWidgetTitle": "Insights Overview",
-        "TitleConsideredInsightsChanges": "The rows increased or decreased by at least %1$s visits (%2$s%% of %3$s total visits).",
-        "TitleConsideredInsightsGrowth": "The following rows have a growth of at least %1$s%% compared to %2$s.",
-        "TitleConsideredMoversAndShakersChanges": "Considered movers only if they grew by more than %1$s%% visits or shrank by less than %2$s%% visits, new entries only if they increased by more than %3$s%% visits (%4$s), and disappeared rows if they shrank by less than %5$s%% visits (%6$s).",
-        "TitleConsideredMoversAndShakersGrowth": "%1$s changed from %2$s to %3$s compared to %4$s. Based on this an evolution of each row of %5$s%% is expected.",
-        "TitleRowChangeDetails": "'%1$s' changed from %2$s (%3$s) to %4$s (%5$s) %6$s.",
-        "TitleRowDisappearedDetails": "'%1$s' decreased by %2$s and disappeared in %3$s compared to %4$s.",
-        "TitleRowMoverAndShaker": "This row had a higher impact than the average.",
+        "TitleConsideredInsightsChanges": "These rows increased or decreased by at least %1$s visits (%2$s%% of %3$s total visits).",
+        "TitleConsideredInsightsGrowth": "These rows grew at least %1$s%% compared to %2$s.",
+        "TitleConsideredMoversAndShakersChanges": "Considered movers only with growth of more than %1$s%% visits or falling by more than %2$s%%. As new entries only if increasing by more than %3$s%% (%4$s), and disappeared rows if shrinking by fewer than %5$s%% (%6$s).",
```

```suggestion
        "TitleConsideredMoversAndShakersChanges": "Movers have more growth than %1$s%% of entries, or fall by more than the bottom %2$s%% did.\nEverything below 5%% of the total amount of visits (%5$s%% individually, which is %6$s visits within the period) disappears, and appears as new entries only and every time they make it above this threshold (by growing more than %3$s%% individually, which is %4$s visits within the period).",
```
Does away with "percentile".

---

For me that doesn't sound like the same before. 

> Movers grow more in a period than %1$s%% of entries did, or shrink more than the %2$s%% falling the most.

The original text said something about `growing more than X% visits`. Isn't that something else than `growing more than X% of entries did`?

---

@sgiehl True, but the X% isn't a number of visits, it is a percentage, which is actually a percentile gauge from what I remember.
(Ref. [the discussion on the original string above](https://github.com/matomo-org/matomo/pull/19437/files#r909352133))
To get away from the idea of percentiles, "growth" in this suggestion implies something ultimately ultimately measured in page-visits. (This is clear from reading "visits" later.)
It is however clear that it is in the upper (and lower) echelon from "than then ___ falling the most___.

From my reply in that original discussion I had it figured out at some point.